core/templates/dev/head/pages/base.html

```diff
@@ -119,6 +119,10 @@
       {% endif %}
     </script>
 
+    <!-- Note to developers: This script needs to be loaded before external
```

Comment:
I actually think I prefer a single JS file, since we're planning on eventually removing base.html. Without base.html, we would need to duplicate this in many places. I really prefer readability and maintainability over efficiency here.